source/api_reference/version-history/index.html.md.erb

```diff
@@ -39,5 +41,13 @@ These changes mean that:
 - `blob-hash` is computed differently from the existing `item-hash` vaues, so if your code relies on [updates](/getting_updates), you should fetch all the data again after updating to this version of the API
 - the field `index-entry-number` no longer exists (use `entry-number` instead)
 - `/entries/{entry-number}` now returns a single object instead of an array of objects
-- `/records/{key}` now returns a flat record object with the entry key using the reserved field `_id`,  instead of an object indexed by the entry key. In addition the `entry-number` and `entry-timestamp` have been removed from the object. If you require these you can get them using the `/records/{key}/entries` endpoint.  
-- `/records` now returns an array of record objects, instead of an object indexed by entry key
\ No newline at end of file
+- `/records/{key}` now returns a flat record object with the entry key using the reserved field `_id`,  instead of an object indexed by the entry key. In addition the `entry-number` and `entry-timestamp` have been removed from the object. If you require these you can get them using the `/records/{key}/entries` endpoint.
+- `/records` now returns an array of record objects, instead of an object indexed by entry key
+
+### Added `/blobs`
+You can now use the `/blobs` endpoint to retrieve multiple blobs in a single request. This is faster than fetching blobs one by one when using the `/entries` endpoint.
+
+`/blobs` is paginated, and by default it returns the first 100 blobs in the register. To fetch more data, query the URL provided in the `Link` header that has `rel=next`.
+
+If you need to incrementally update a copy of the register, you can store the last URL you queried, and query it again to check for updates. You do not need to fetch earlier pages
```

Comment:
Is this last paragraph trying to cover the fact that blobs are ordered and so pages never change? If this is the case I would rather say that instead of providing a solution.

Reply:
Yes - I will try and reword this. I do want to explain a bit why the endpoint is useful though